cli/src/config/templates.toml

```diff
@@ -281,9 +288,39 @@ separate(" ",
         if(commit.git_head(), label("git_head", "git_head()")),
         format_short_commit_id(commit.commit_id()),
         if(commit.conflict(), label("conflict", "conflict")),
+        if(config("ui.show-cryptographic-signatures").as_boolean(),
+          format_short_cryptographic_signature(commit.signature())),
       )
 '''
 
+# signature verification is slow, disable by default
```

Comment:
nit: maybe this comment should be attached to `ui.show-cryptographic-signatures`?